shaders/src/light_indirect.fs

```diff
@@ -595,6 +602,16 @@ void evaluateIBL(const MaterialInputs material, const PixelParams pixel, inout v
     Fr = isEvaluateSpecularIBL(pixel, shading_normal, shading_view, shading_NoV);
 #endif
 
+#if defined(HAS_REFLECTIONS) && REFLECTIONS_MODE == REFLECTIONS_MODE_SCREEN_SPACE
+    // Up to this point Fr is the Cubemap reflection only.
+    // evaluateScreenSpaceReflections will replace the value of Fr if there's a hit.
+    // TODO: do we want iblLuminance to control screen-space reflections?
+    if (pixel.roughness <= 0.01f && frameUniforms.ssrEnabled > 0u) {
```

Comment:
Should we optimize by only evaluating the IBL when there's no hit instead?

Reply:
That's a good idea for now, but I think with blurry reflections we'll want to sample both anyway. I'm not sure yet.